profiles/realtime-virtual-guest/tuned.conf

```diff
@@ -25,6 +25,7 @@ assert2=${f:assertion:isolated_cores contains online CPU(s):${isolated_cores_exp
 # group.group_name=rule_priority:scheduler_policy:scheduler_priority:core_affinity_in_hex:process_name_regex
 # for i in `pgrep ksoftirqd` ; do grep Cpus_allowed_list /proc/$i/status ; done
 group.ksoftirqd=0:f:2:*:^\[ksoftirqd
+group.ktimers=0:f:2:*:^\[ktimers
```

Comment:
I'm not sure what the `# for i in ...` comment above is there for so I've just squashed the new kthread priority assignment against ksoftirqd. If the comment is a required convention, I'll change that.

Reply:
It seems these comments originated from patch provided by Jeremy Eder <jeder@redhat.com> and Clark Williams <williams@redhat.com>. I guess it's just helper to remember how to debug/verify CPU affinity for the rule. We could probably drop them.